app/scripts/Xprize-VisualRetargeting.xml

```diff
@@ -19,26 +19,27 @@
   </module>
 
   <module>
-    <name>camCalibWithPose</name>
+    <name>camCalib</name>
     <parameters>--context cameraCalibration --from icubEyes.ini --group CAMERA_CALIBRATION_LEFT --name /icub/camcalib/left</parameters>
     <node>icub-console</node>
   </module>
 
   <module>
-    <name>camCalibWithPose</name>
+    <name>camCalib</name>
     <parameters>--context cameraCalibration --from icubEyes.ini --group CAMERA_CALIBRATION_RIGHT --name /icub/camcalib/right</parameters>
     <node>icub-console</node>
   </module>
 
   <module>
     <name>yarpdev</name>
-    <parameters>--device JoypadControlServer --use_separate_ports 1 --period 10 --name /joypadDevice/Oculus --subdevice openxrheadset --stick_as_axis true --gui_elements 1 --GUI_0::width 0.15 --GUI_0::height 0.15 --GUI_0::x 0.2 --GUI_0::y -0.2 --GUI_0::z -0.5</parameters>
+    <parameters>--device JoypadControlServer --use_separate_ports 1 --period 10 --name /joypadDevice/Oculus --subdevice openxrheadset --stick_as_axis true --gui_elements 1 --GUI_0::width 0.15 --GUI_0::height 0.15 --GUI_0::x 0.15 --GUI_0::y -0.2 --GUI_0::z -0.5</parameters>
```

Comment:
Probably, here we can use `--from dcmWalkingRetargeting.ini ` instead of this long line of list of parameters.
Should be checked, if all the variables are correctly set then in the `.ini` file.